scripts/up.sh

```diff
@@ -50,7 +50,7 @@ function pod_cidrs() {
 
 function add_routes() {
   unset IFS
-  routes=$(kubectl --kubeconfig ${2} get nodes -o jsonpath='{range .items[*]}ip route add {.spec.podCIDR} via {.status.addresses[?(.type=="InternalIP")].address}{"\n"}')
+  routes=$(kubectl --kubeconfig ${2} get nodes -o jsonpath='{range .items[*]}ip route add {.spec.podCIDR} via {.status.addresses[?(.type=="InternalIP")].address}{"\n"}{end}')
```

Comment:
Does this related to https://github.com/kubernetes-sigs/mcs-api/issues/3?

Reply:
I guess so, also get the error: `Command line is not complete. Try option "help"` without the fix.
Not see the #3 until see this comments :D